index.js

```diff
@@ -32,4 +32,29 @@ module.exports = options => options ?
 	getPort(options).catch(() => getPort(Object.assign(options, {port: 0}))) :
 	getPort({port: 0});
 
+module.exports.makeRange = (from, to) => {
+	if (!Number.isInteger(from) || !Number.isInteger(to)) {
+		throw new TypeError('`from` and `to` must ne integer numbers');
+	}
+
+	if (from < 1025 || from > 65535) {
```

Comment:
Should not throw when `from` is equal to 1024.

Reply:
Huh, that's peculiar. I always though restricted range was length 1024(0 isn't a port), but it turns out it's actually length 1023, thanks for the info.